redbot/cogs/general/general.py

```diff
@@ -267,7 +267,11 @@ async def serverinfo(self, ctx, details: bool = False):
             data.add_field(name=_("Voice Channels"), value=voice_channels)
             data.add_field(name=_("Roles"), value=humanize_number(len(guild.roles)))
             data.add_field(name=_("Owner"), value=str(guild.owner))
-            data.set_footer(text=_("Server ID: ") + str(guild.id))
+            data.set_footer(
+                text=_("Server ID: ")
+                + str(guild.id) 
+                + _(" | Use `{prefix}serverinfo t` for more info on the server.")
```

Comment:
You're missing a ``.format(prefix=ctx.prefix)`` at the end.

Reply:
`clean_prefix` 😅 we don't want the bot to ping stuff.